5_databases/week1/homework

```diff
@@ -0,0 +1,55 @@
+-- Active: 1699216915341@@0.0.0.0@3306@hyf_lesson1
+
+SELECT COUNT(*)
+ FROM task;
+
+
+
+
+SELECT COUNT(*) 
+FROM task 
+WHERE due_date <= CURDATE();
```

Comment:
This is an awesome interpretation, that could actually be a real life solution. These tasks are from 2017 tho, so all of them are basically expired. Eventually just try just to look for what is NULL here to practice the `WHERE field IS NULL` statement